src/main/kotlin/org/rust/ide/inspections/lints/RsUnusedMustUseInspection.kt

```diff
@@ -0,0 +1,106 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections.lints
+
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.openapi.project.Project
+import com.intellij.codeInspection.LocalQuickFix
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.RsBundle
+import org.rust.ide.annotator.getFunctionCallContext
+import org.rust.ide.inspections.RsProblemsHolder
+import org.rust.ide.utils.template.newTemplateBuilder
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.*
+import org.rust.lang.core.types.implLookupAndKnownItems
+import org.rust.lang.core.types.ty.TyAdt
+import org.rust.lang.core.types.type
+import org.rust.openapiext.createSmartPointer
+
+private fun RsExpr.returnsStdResult(): Boolean {
+    val (_, knownItems) = implLookupAndKnownItems
+    val type = type as? TyAdt ?: return false
+    return type.item == knownItems.Result
+}
+
+private class FixAddLetUnderscore : LocalQuickFix {
+    override fun getFamilyName() = RsBundle.message("inspection.UnusedMustUse.FixAddLetUnderscore.name")
+
+    override fun applyFix(project: Project, descriptor: ProblemDescriptor) {
+        val expr = descriptor.psiElement as RsExpr
+        expr.parent.replace(RsPsiFactory(project).createLetDeclaration("_", expr))
```

Comment:
Maybe a template builder could be used here to select `_` and let the user change the binding? Same as is done with in `FixAddExpect`.